src/main/scala/temple/generate/target/openapi/OpenAPIBuilder.scala

```diff
@@ -0,0 +1,129 @@
+package temple.generate.target.openapi
+
+import io.circe.syntax._
+import io.circe.yaml.Printer
+import temple.DSL.semantics.AttributeType._
+import temple.DSL.semantics.{Annotation, Attribute}
+import temple.collection.FlagMapView
+import temple.generate.Endpoint._
+import temple.generate.target.openapi.OpenAPIFile.{Components, Info}
+import temple.generate.target.openapi.OpenAPIType._
+
+import scala.collection.immutable.ListMap
+import scala.collection.mutable
+
+private class OpenAPIBuilder private (name: String, version: String, description: String = "") {
```

Comment:
I'm not sure `Builder` is the right term - we've been using `Generator` everywhere else (e.g. `PostgresGenerator`, `DockerfileGenerator`), and leaving `Builder` for when we do the mapping?